lib/faker.ex

```diff
@@ -126,11 +126,12 @@ defmodule Faker do
     quote do
       def unquote(function)() do
         module = Module.concat(unquote(__CALLER__.module), Faker.mlocale())
+        module_enus = Module.concat(unquote(__CALLER__.module), Faker.mlocale())
         cond do
           function_exported?(module, unquote(function), 0) ->
             apply(module, unquote(function), [])
-          function_exported?(module, unquote(function), EnUs) ->
-            apply(Module.concat(unquote(__CALLER__.module), EnUs), unquote(function), [])
+          function_exported?(module_enus, unquote(function), 0) ->
```

Comment:
I don't see the difference between this and the previous cond branch. I don't think it fixes the problem, just removes the feature. The idea of this one is to apply the default country.

Reply:
The third parameter to `Kernel.function_exported?` is the arity of the function. `EnUs` is not arity. Instead, the module name should be concatenated with `EnUs`, and then check whether the function with arity `0` is exported.

Reply:
Note that it's checking `module_enus` not `module`

Reply:
Oops, my bad. I should be concatenating it with `EnUs` not `Faker.mlocale`

Reply:
Fix pushed!